docs/user_guide/configuring.rst

```diff
@@ -34,8 +34,45 @@ If you'd like it to link to another page or use an external link instead, use th
 Configure icon links
 ====================
 
-If you'd like icon links to show up to the right of your main navigation bar, use the
-following configuration:
+You can add icon links to show up to the right of your main navigation bar.
+
+These links take the following form:
+
+.. code:: python
+
+   html_theme_options = {
+       ...
+       "icon_links": [
+           {
+               # Label for this link
+               "name": "GitHub",
+               # URL where the link will redirect
+               "url": "https://github.com/<your-org>/<your-repo>",  # required
+               # Icon class (if type=='fontawesome'), or path to local image (if type=='local')
```

Comment:
```suggestion
               # Icon class (if "type": "fontawesome"), or path to local image (if "type": "local")
```
Do not use python syntax because it can be confusing...